app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -259,7 +259,15 @@ private void search() {
                     progressBar.setVisibility(View.GONE);
                     this.results = results.first;
                     adapter.updateItems(results.first);
-                    adapterFeeds.updateData(results.second);
+                    if(getArguments().getLong(ARG_FEED,0)==0)
```

Comment:
For AntennaPod, we use a slightly different code style. It is based on the [Google Java code style](https://google.github.io/styleguide/javaguide.html). You can find details about what should be changed on the "checks" tab on GitHub.

Reply:
I went through the "checks" tab and have found the issue spotted by CircleCI Checks / static-analysis. Fixing it right away.

Just to be clear, you were talking about that particular error only, right?

Reply:
Yes, the `static-analysis` checks that the code style matches Google's style.